engine/src/main/java/org/terasology/logic/ai/HierarchicalAISystem.java

```diff
@@ -125,17 +125,17 @@ private void loop(EntityRef entity, LocationComponent location,
 
             // if our AI is aggressive or hunter go and hunt player else run away
             // if wild
-            if (ai.aggressive) {
+            /*if (ai.aggressive) {
```

Comment:
At this point, CheckStyle was complaining about an empty block.  The code was:
`if (ai.aggressive) {
}`